src/main/resources/templates/fragments/cardHeader.html

```diff
@@ -1,9 +1,13 @@
 <header
     th:fragment="cardHeader"
-    th:with="hasSubtext=${!#strings.isEmpty(subtext)}"
+    th:with="hasSubtext=${!#strings.isEmpty(subtext)},
+             requiredField=${required ?: false}"
     th:assert="${!#strings.isEmpty(header)}"
     class="form-card__header">
-  <h1 id="header" class="h2" th:utext="${header}"></h1>
+  <h1 id="header" class="h2">
+    <span th:text="${header}"></span>
```

Comment:
Did you mean to take out `utext` here?   I'm going to add it back in unless it was intentional.

Reply:
We can leave it in